be/src/storage/task/engine_storage_migration_task.cpp

```diff
@@ -235,6 +251,51 @@ Status EngineStorageMigrationTask::_storage_migrate(TabletSharedPtr tablet) {
             break;
         }
 
+        if (tablet->updates() == nullptr) {
+            res = _finish_migration(tablet, end_version, shard, consistent_rowsets, new_tablet_meta, schema_hash_path,
+                                    new_meta_file, dcgs_snapshot_path, need_remove_new_path);
+        } else {
+            res = _finish_primary_key_migration(tablet, end_version, shard, consistent_rowsets, schema_hash_path,
+                                                new_meta_file, need_remove_new_path);
+        }
+    } while (false);
+
+    // 4. clear
+    if (!new_meta_file.empty()) {
+        // remove meta file
+        Status st = fs::remove(new_meta_file);
+        if (!st.ok()) {
+            LOG(WARNING) << "failed to remove meta file. tablet_id=" << _tablet_id << ", schema_hash=" << _schema_hash
+                         << ", path=" << schema_hash_path << ", error=" << st.to_string();
+        }
+    }
+    if (!dcgs_snapshot_path.empty()) {
+        // remove dcg snapshot file
+        Status st = fs::remove(dcgs_snapshot_path);
+        if (!st.ok()) {
+            LOG(WARNING) << "failed to remove dcg file. tablet_id=" << _tablet_id << ", schema_hash=" << _schema_hash
+                         << ", path=" << schema_hash_path << ", error=" << st.to_string();
+        }
+    }
+    if (!res.ok() && need_remove_new_path) {
+        // remove all index and data files if migration failed
+        Status st = fs::remove_all(schema_hash_path);
+        if (!st.ok()) {
```

Comment:
Output the `tablet_id` in the error log.

Reply:
done